Author: Sunil Mohan Adapa <sunil@medhas.org>
Description: Debian patches found in version 0.5.8e-1.
 Originally the patches were maintained in git, but in version 0.5.8e-2
 they were migrated to individual patches in d/patches/ instead.

 Originates from the following git commits:

commit 240542eccf4f0252c3d7d779d83bc47136a1bbd6
Date:   Sat Mar 15 23:07:44 2014 +0530

    Maintain debian native scripts in deb/ instead of debian/
    
    It has been suggested that in order to upload pagekite to Debian
    it would be easy if the non-native packages be built by Debian team
    while native package building stays separate in upstream. To
    maintain non-native package changes easily, keeping upstream debian
    scripts in deb/ directory would greately help.

commit 30c3b869005de6d238f08d69a3d6e4611cc2877d
Date:   Sat Mar 15 21:34:40 2014 +0530

    Autobuild incremented .deb version when version is incremented

commit f64e5e28a8d703f7e60a9f73fff3b2a87e563fcc
Date:   Sat Mar 15 20:09:06 2014 +0530

    Switch to pybuild in debhelper, remove need to delete setup.cfg

commit bb550b7b3cf82a3b14dc1c605dfd7fd1a01cf11b
Date:   Sat Mar 15 07:29:37 2014 +0530

    Cleanup man page building for debian packages

commit 83c7bda06191bd73060f6bfec130c42d34857941
Date:   Fri Mar 14 20:11:50 2014 +0530

    Clean debian files using debuild

commit ed3a0073568c580617d40a86fd423a342b5e3427
Date:   Fri Mar 14 20:10:04 2014 +0530

    Fix Makefile regressed dependency for creating debian package

commit 0102762429c99d408a39a31318b6bbe030aba3fa
Date:   Fri Mar 14 19:37:59 2014 +0530

    Build debian source package also

commit 802b5e86e011e9fa7e67b4b83af6230cf195a01b
Date:   Fri Mar 14 19:16:28 2014 +0530

    Dont auto generate changelog, remove version from package name

commit 2155cc92b711c8e0fefe448f769517203f0f9380
Date:   Fri Mar 14 07:56:35 2014 +0530

    Copy logrotate file to debian directory

commit 474f2cf94e6007fa57ba57fe9143114a11d76acd
Date:   Fri Mar 14 07:51:28 2014 +0530

    Remove need for generating debian/control file

commit 55d05376734e4c12555bbf6522ee28ef93c6cbf6
Date:   Fri Mar 14 07:41:41 2014 +0530

    Remove need for generating debian/copyright file

Index: pagekite-alioth/Makefile
===================================================================
--- pagekite-alioth.orig/Makefile	2016-09-12 11:00:50.921803994 +0000
+++ pagekite-alioth/Makefile	2016-09-12 11:00:56.469868556 +0000
@@ -88,28 +88,21 @@
 	                           --requires=python-SocksipyChain
 
 VERSION=`python setup.py --version`
-.debprep: doc/pagekite.1
-	@rm -f setup.cfg
-	@sed -e "s/@VERSION@/$(VERSION)/g" \
-		< debian/control.in >debian/control
-	@sed -e "s/@VERSION@/$(VERSION)/g" \
-		< debian/copyright.in >debian/copyright
-	@sed -e "s/@VERSION@/$(VERSION)/g" \
-	     -e "s/@DATE@/`date -R`/g" \
-		< debian/changelog.in >debian/changelog
-	@ls -1 doc/*.? >debian/pagekite.manpages
-	@ln -fs ../etc/logrotate.d/pagekite.debian debian/pagekite.logrotate
-	@ln -fs ../etc/init.d/pagekite.debian debian/init.d
+DEB_VERSION=`head -n1 debian/changelog | sed -e "s+.*(\(.*\)).*+\1+"`
+.debprep:
+	@ln -sf deb debian
+	if [ "x$(VERSION)" != "x$(DEB_VERSION)" ] ; \
+	then \
+	  dch --maintmaint --newversion $(VERSION) --urgency=low \
+              --distribution=unstable "New release." ; \
+	fi
 
-.targz: .debprep
+.targz:
 	@python setup.py sdist
 
-.deb: .targz
-	@cp -v dist/pagekite*.tar.gz \
-		../pagekite-$(VERSION)_$(VERSION).orig.tar.gz
-	@debuild -i -us -uc -b
+.deb: .debprep
+	@debuild -i -us -uc
 	@mv ../pagekite_*.deb dist/
-	@rm ../pagekite-*.orig.tar.gz
 
 .header: pagekite doc/header.txt
 	@sed -e "s/@VERSION@/$(VERSION)/g" \
@@ -151,6 +144,6 @@
 	@rm -vf sockschain *.pyc */*.pyc */*/*.pyc scripts/breeder.py .SELF
 	@rm -vf .appver pagekite-tmp.py MANIFEST setup.cfg pagekite_gtk.py
 	@rm -vrf *.egg-info .header doc/pagekite.1 build/
-	@rm -vf debian/files debian/control debian/copyright debian/changelog
-	@rm -vrf debian/pagekite* debian/python* debian/init.d
+	-debuild clean
+	@-rm debian
 
